_posts/2024-02-26-cache-refresh.md

```diff
@@ -0,0 +1,166 @@
+        <img src="{{site.url}}/assets/images/logos/delta-lake.png" title="Delta Lake connector">
+      </a>
+    </div>
+    <div class="col-sm">
+      <a href="{{site.url}}/docs/current/connector/hudi.html" target="_blank">
+        <img src="{{site.url}}/assets/images/logos/apache-hudi.png" title="Hudi connector">
+      </a>
+    </div>
+    <div class="col-sm">
+      <a href="{{site.url}}/docs/current/connector/iceberg.html" target="_blank">
+        <img src="{{site.url}}/assets/images/logos/apache-iceberg.png" title="Iceberg connector">
+      </a>
+    </div>
+  </div>
+</div>
+
+Hive is still around, just like HDFS, but we consider them both close to legacy
+status. Yet all four connectors could benefit from caching. Good news came at
+Trino Summit 2022 when Hope Wang and Beinan Wang from
+[Alluxio]({{site.url}}/ecosystem/add-on.html#alluxio) presented about their
+integration with Trino and the Hive connector - [Trino optimization with
+distributed caching on data lake]({% post_url
+2023-07-21-trino-fest-2023-alluxio-recap %}). They mentioned plans to open
+source their implementation and an initial PR was created.
+
+<div class="container">
+  <div class="row">
+    <div class="col-sm"></div>
+    <div class="col-sm">
+      <img src="{{site.url}}/assets/images/logos/alluxio.png" title="Alluxio">
+    </div>
+    <div class="col-sm"></div>
+  </div>
+</div>
+
+## Collaboration
+
+The initial presentation and pull request (PR) planted a seed in the community.
+The Trino project had been moving fast in terms of deprecating the old
+dependencies from the Hadoop and Hive ecosystem so the Alluxio PR was not ready
+for a merge. Discussions with [David Phillips](https://github.com/electrum) laid
+out the path to adjust to the new file system support.
+
+In the end it was [Florent Delannoy](https://github.com/pluies) who started
+another [PR for file system caching support, specifically for the Delta Lake
+connector](https://github.com/trinodb/trino/pull/18719). His team mate [Jonas
+Irgens Kylling](https://github.com/jkylling), also a [presenter from Trino Fest
+  2023]({% post_url 2023-07-14-trino-fest-2023-dune %}), took over the work on
+the PR. The collaboration on it was an **epic effort**. Over 300
+comments directly on GitHub and numerous hours of coding, reviewing, testing,
+and discussion on slack and elsewhere finally ended with a merge after many
+months.
+
+Special props for their help for Florent and Jonas must go out to [David
+Phillips](https://github.com/electrum), [Raunaq
+Morarka](https://github.com/raunaqmorarka), [Piotr
+Findeisen](https://github.com/findepi), [Mateusz
+Gajewski](https://github.com/wendigo), [Beinan Wang](https://github.com/beinan),
+[Amogh Margoor](https://github.com/amoghmargoor), and [Marton
+Bod](https://github.com/marton-bod).
+
+## Finishing
+
+In parallel to the work on the initial PR for Delta Lake, I ended working on the
```

Comment:
```suggestion
In parallel to the work on the initial PR for Delta Lake, I ended up working on the
```

Reply:
Maybe say "yours truly" instead of "I," which is a little jarring for a blog that isn't otherwise written in first person.

Reply:
sounds good